rotkehlchen/chain/aggregator.py

```diff
@@ -972,25 +980,30 @@ def query_eth_balances(
         self.query_evm_chain_balances(chain=SupportedBlockchain.ETHEREUM)
         self.query_defi_balances()
         self._add_eth_protocol_balances(eth_balances=self.balances.eth)
-        self._add_staking_protocol_balances()
+        self._add_staking_protocol_balances(chain_id=ChainID.ETHEREUM)
 
-    def _add_staking_protocol_balances(self) -> None:
+    def _add_staking_protocol_balances(self, chain_id: CHAIN_IDS_WITH_BALANCE_PROTOCOLS) -> None:  # noqa: E501
```

Comment:
you still did not change the name as asked in the previous comment: https://github.com/rotki/rotki/pull/6638#discussion_r1336947569

```suggestion
    def _query_staking_protocol_balances(self, chain_id: CHAIN_IDS_WITH_BALANCE_PROTOCOLS) -> None:  # noqa: E501
```